packages/suite/src/components/wallet/TransactionItem/index.tsx

```diff
@@ -10,41 +11,92 @@ import { isTestnet } from '@wallet-utils/accountUtils';
 import { isTxUnknown } from '@wallet-utils/transactionUtils';
 import { AccountMetadata } from '@suite-types/metadata';
 import { WalletAccountTransaction } from '@wallet-types';
-// local
 import TransactionTypeIcon from './components/TransactionTypeIcon';
 import TransactionHeading from './components/TransactionHeading';
 import { MIN_ROW_HEIGHT } from './components/BaseTargetLayout';
 import { Target, TokenTransfer, FeeRow } from './components/Target';
 import TransactionTimestamp from './components/TransactionTimestamp';
+import { useAnchor } from '@suite-hooks/useAnchor';
+import { AccountTransactionBaseAnchor } from '@suite-constants/anchors';
+import { SECONDARY_PANEL_HEIGHT } from '@suite/components/suite/AppNavigation';
 
-const Wrapper = styled.div<{ chainedTxMode?: boolean }>`
+const Wrapper = styled.div<{
+    chainedTxMode?: boolean;
+    isPending?: boolean;
+    shouldHighlight?: boolean;
+}>`
     display: flex;
     flex-direction: row;
-    padding: 12px 0px;
+    padding: 0 24px;
+    background: ${props => props.theme.BG_WHITE};
+    transition: all 0.3s;
+    transition-delay: 0.3s;
+    outline-offset: -3px;
+
+    /* height of secondary panel and a gap between transactions and graph */
+    scroll-margin-top: calc(${SECONDARY_PANEL_HEIGHT} + 115px);
+
+    &:first-of-type {
+        border-top-left-radius: 12px;
+        border-top-right-radius: 12px;
+    }
+
+    &:last-of-type {
+        border-bottom-left-radius: 12px;
+        border-bottom-right-radius: 12px;
+    }
 
     ${props =>
         props.chainedTxMode
             ? css`
                   width: 100%;
-                  padding: 12px 16px;
+                  padding: 0px 40px;
                   cursor: pointer;
                   &:hover {
                       border-radius: 8px;
                       background: ${props => props.theme.BG_GREY};
                   }
               `
             : css`
-                  & + & {
-                      border-top: 1px solid ${props => props.theme.STROKE_GREY};
+                  &:not(:first-child) {
+                      > * {
+                          border-top: 1px solid ${props => props.theme.STROKE_GREY};
+                      }
                   }
               `}
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        padding: 0px 16px;
+    }
+
+    ${props =>
+        props.isPending &&
+        css`
+            border-left: 8px solid ${props => props.theme.TYPE_ORANGE};
+            padding-left: 14px;
+
+            @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+                padding: 0px 10px;
+            }
+        `}
+
+    ${props =>
+        props.shouldHighlight &&
```

Comment:
Would be nice to define those anchor-related styles that should remain consistent on one place.  I could imaginge outline, background, border radiuses, transitions...